src/olympia/lib/settings_base.py

```diff
@@ -1665,6 +1665,10 @@ def get_redis_settings(uri):
 
     # We don't allow refreshes, instead we simply have a long duration.
     'JWT_ALLOW_REFRESH': False,
+
+    # Prefix for non-apikey jwt tokens. Should be different from 'JWT' which we
+    # already used for api key tokens.
+    'JWT_AUTH_HEADER_PREFIX': 'JWT-i',
```

Comment:
We have to keep `JWT` for api-key based auth, for backwards-compatibility, so I could use `Bearer` for the frontend token. What do you think ?

I wanted the clear separation between the 2 to keep things fast. My goal is to include the 2 auth classes on most of our API views, which would mean DRF would go through both authentication classes for each request. I'd like to avoid wasting time when it does that, and bail as early as possible instead of decoding payloads.